File: Test/AddDefineResultTest.java
package test;

import static org.junit.Assert.*;

import org.junit.Test;

import com.lucciola.result.AddDefineResult;
import com.lucciola.result.CommunicationResult;

public class AddDefineResultTest {

	@Test
	public void test() {
		CommunicationResult result = new AddDefineResult("{\"AddedDefine\":\"pluse = 1 + 1\"}");
		assertEquals(result.getMessage(), "pluse = 1 + 1");
	}

}

File: src/test/AddDefineResultTest.kt
package test

import org.junit.Assert.*

import org.junit.Test

import com.lucciola.result.AddDefineResult
import com.lucciola.result.CommunicationResult

class AddDefineResultTest {

    @Test
    fun test() {
        val result = AddDefineResult("{\"AddedDefine\":\"pluse = 1 + 1\"}")
        assertEquals(result.message, "pluse = 1 + 1")
    }

}
